intersections/build.boot

```diff
@@ -0,0 +1,113 @@
+            dev-file (io/file tmp (str dev-dir dev-name))]
+        (u/dbug* "Minifying content of: %s\n" real-dir)
+        (let [uglify-result (minify-js/minify-js (mapv #(str real-dir %) file-name)
+                                                 prod-file)]
+          (u/dbug* "Uglify result: %s\n" (u/pp-str uglify-result)))
+        (u/dbug* "Minified file: %s\n" prod-file)
+        (u/dbug* "Development file: %s\n" dev-file)
+        (io/make-parents dev-file)
+        (file/copy-atomically prod-file dev-file)
+        (-> fileset (c/add-resource tmp) c/commit!)))))
+
+(deftask package []
+  (let [uglify-dir (normalize-path "uglify/")
+        prod-dir (normalize-path "cljsjs/intersections/production/")
+        dev-dir (normalize-path "cljsjs/intersections/development/")]
+    (comp
+     (download :url "https://raw.githubusercontent.com/thelonious/kld-intersections/development/lib/Intersection.js"
+               :checksum "2FF91BB218A7318DA4F7625A8DCFD27C")
+     (download :url "https://raw.githubusercontent.com/thelonious/kld-intersections/development/lib/IntersectionParams.js"
+               :checksum "6C7979D5AD687F729138DFDE61BD3330")
+     (sift :move {#"^Intersection.js" (str uglify-dir "Intersection.js")})
+     (sift :move {#"^IntersectionParams.js" (str uglify-dir "IntersectionParams.js")})
+
+     ;; dep kld-affine
+     (download :url "https://raw.githubusercontent.com/thelonious/kld-affine/development/lib/Point2D.js"
+               :checksum "3220937049C74FA759C83BF61A602924")
+     (download :url "https://raw.githubusercontent.com/thelonious/kld-affine/development/lib/Vector2D.js"
+               :checksum "049E2B087DFA2C0893CF0FB3795F8842")
+     (download :url "https://raw.githubusercontent.com/thelonious/kld-affine/development/lib/Matrix2D.js"
+               :checksum "D3FDD36BD18578B2A7DAE3CD62206CC4")
+     (sift :move {#"^Matrix2D.js" (str uglify-dir "Matrix2D.js")})
+     (sift :move {#"^Vector2D.js" (str uglify-dir "Vector2D.js")})
+     (sift :move {#"^Point2D.js" (str uglify-dir "Point2D.js")})
+
+     ;; dep kld-polynomial
+     (download :url "https://raw.githubusercontent.com/thelonious/kld-polynomial/development/lib/Polynomial.js"
+               :checksum "BE6181D3332E43A3A74BC7EA53DEFBF6")
+     (download :url "https://raw.githubusercontent.com/thelonious/kld-polynomial/development/lib/PolynomialAdditionalMethods.js"
+               :checksum "A6F11C8AF14580BD88D17214AE2292F9")
+     (download :url "https://raw.githubusercontent.com/thelonious/kld-polynomial/development/lib/SqrtPolynomial.js"
+               :checksum "10CDC2F3F311082711B8A92F6B33300C")
+     (sift :move {#"^Polynomial.js" (str uglify-dir "Polynomial.js")})
+     (sift :move {#"^PolynomialAdditionalMethods.js" (str uglify-dir "PolynomialAdditionalMethods.js")})
+     (sift :move {#"^SqrtPolynomial.js" (str uglify-dir "SqrtPolynomial.js")})
+
+     (uglify :file-name ["Point2D.js"
```

Comment:
I not huge fan of this approach, as the packages also have `index.js` that exposes everything, it should be possible to use browerify/webpack to concat and minify these. But I can also accept this solution.